### PR TITLE
azure-ai-evaluation README update

### PR DESCRIPTION
**Pull request by singankit**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by w-javed — APPROVED** [review]

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+- Evaluate existing data from generative ai applications
+- Evaluate generative ai applications
+- Evaluate by generating mathematical, AI-assisted quality and safety metrics
+
+Azure AI SDK provides following to evaluate Generative AI Applications:
+- [Evaluators][evaluators] - Generate scores individually or when used together with `evaluate` API.
+- [Evaluate API][evaluate_api] - Python API to evaluate dataset or application using built-in or custom evaluators.
+
+
+
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/README.md:15]: Remove unnecessary blank lines?

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 2]
```diff
@@ -20,88 +32,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
...
 
 ## Key concepts
 
-Evaluators are custom or prebuilt classes or functions that are designed to measure the quality of the outputs from language models.
+## Evaluators
+
+Evaluators are custom or prebuilt classes or functions that are designed to measure the quality of the outputs from language models or generative ai applications.
 
-## Examples
+### Built-in evaluators
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/README.md:47]: Built-in Evaluators?

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 3]
```diff
@@ -20,88 +32,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
...
 
-### Evaluators
+Built-in evaluators are out of box evaluators provided by Microsoft:
+| Category  | Evaluator class                                                                                                                    |
+|-----------|------------------------------------------------------------------------------------------------------------------------------------|
+| [Performance and quality](#performance-and-quality-evaluators) (AI-assisted)  | `GroundednessEvaluator`, `RelevanceEvaluator`, `CoherenceEvaluator`, `FluencyEvaluator`, `SimilarityEvaluator`, `RetrievalEvaluator` |
+| [Performance and quality](#performance-and-quality-evaluators) (NLP)  | `F1ScoreEvaluator`, `RougeScoreEvaluator`, `GleuScoreEvaluator`, `BleuScoreEvaluator`, `MeteorScoreEvaluator`|
+| [Risk and safety](#risk-and-safety-evaluators ) (AI-assisted)    | `ViolenceEvaluator`, `SexualEvaluator`, `SelfHarmEvaluator`, `HateUnfairnessEvaluator`, `IndirectAttackEvaluator`, `ProtectedMaterialEvaluator`                                             |
+| [Composite](#composite-evaluators) | `QAEvaluator`, `ContentSafetyEvaluator`                                             |
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/README.md:55]: How about multimodal ones?

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 4]
```diff
@@ -20,88 +32,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
...
+Built-in evaluators are great out of the box to start evaluating your application's generations. However you can build your own code-based or prompt-based evaluator to cater to your specific evaluation needs.
 
+```python
 
+# Custom evaluator as a function to calculate response length
 def response_length(response, **kwargs):
-    return {"value": len(response)}
+    return len(response)
 
+# Custom class based evaluator to calculate answer length
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/README.md:108]: answer -> response

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 5]
```diff
@@ -1,16 +1,28 @@
 # Azure AI Evaluation client library for Python
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/README.md:1]: "client library" -> "SDK"?

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 6]
```diff
@@ -20,88 +32,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
...
+            } 
+        }
+    }
+)
 ```
+Above code snippet refers to askwiki application in this [sample][evaluate_app].
+
+For more details refer to [Evaluate on a target][evaluate_target]
+
 ### Simulator
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/README.md:181]: For the TSG section below, could we replace with https://aka.ms/azureaieval-tsg?

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 7]
```diff
@@ -1,16 +1,28 @@
...
+- Evaluate generative ai applications
+- Evaluate by generating mathematical, AI-assisted quality and safety metrics
+
+Azure AI SDK provides following to evaluate Generative AI Applications:
+- [Evaluators][evaluators] - Generate scores individually or when used together with `evaluate` API.
+- [Evaluate API][evaluate_api] - Python API to evaluate dataset or application using built-in or custom evaluators.
+
+
+
+## Evaluate API
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/README.md:16]: My understanding is this is a overview section. Do we really need a level 2 heading here?

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 8]
```diff
@@ -20,88 +32,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
...
+Install the Azure AI Evaluation library for Python with [pip][pip_link]:
 
 ```bash
 pip install azure-ai-evaluation
 ```
 
 ## Key concepts
 
-Evaluators are custom or prebuilt classes or functions that are designed to measure the quality of the outputs from language models.
+## Evaluators
```
[the remark this answers is not in the document]
Comment (singankit) [reply sdk/evaluation/azure-ai-evaluation/README.md:43]: Done.

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 9]
```diff
@@ -20,88 +32,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
...
-    # Using multiple evaluators together using `Evaluate` API
-
-    result = evaluate(
-        data="evaluate_test_data.jsonl",
-        evaluators={
-            "response_length": response_length,
-            "violence": violence_eval,
-        },
-    )
+### Evaluate Generative AI Application
```
Comment (ninghu) [inline sdk/evaluation/azure-ai-evaluation/README.md:156]: nit: For section titles, I noticed some instances where both words are capitalized and others where only the first word is capitalized. While either style is fine, we should maintain consistency across all titles at least.

File: sdk/evaluation/azure-ai-evaluation/README.md (hunk 6 above)
Comment (singankit) [reply sdk/evaluation/azure-ai-evaluation/README.md:181]: Done

File: sdk/evaluation/azure-ai-evaluation/README.md (hunk 1 above)
Comment (singankit) [reply sdk/evaluation/azure-ai-evaluation/README.md:15]: Done

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 10]
```diff
@@ -20,88 +32,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
...
 
-## Examples
+### Built-in evaluators
 
-### Evaluators
+Built-in evaluators are out of box evaluators provided by Microsoft:
+| Category  | Evaluator class                                                                                                                    |
+|-----------|------------------------------------------------------------------------------------------------------------------------------------|
+| [Performance and quality](#performance-and-quality-evaluators) (AI-assisted)  | `GroundednessEvaluator`, `RelevanceEvaluator`, `CoherenceEvaluator`, `FluencyEvaluator`, `SimilarityEvaluator`, `RetrievalEvaluator` |
+| [Performance and quality](#performance-and-quality-evaluators) (NLP)  | `F1ScoreEvaluator`, `RougeScoreEvaluator`, `GleuScoreEvaluator`, `BleuScoreEvaluator`, `MeteorScoreEvaluator`|
```
[the remark this answers is not in the document]
Comment (singankit) [reply sdk/evaluation/azure-ai-evaluation/README.md:53]: Had a chat offline this looks good.

File: sdk/evaluation/azure-ai-evaluation/README.md (hunk 4 above)
Comment (singankit) [reply sdk/evaluation/azure-ai-evaluation/README.md:108]: This is done

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 11]
```diff
@@ -20,88 +26,152 @@ For a more complete set of Azure libraries, see https://aka.ms/azsdk/python/all
```
Comment (needuv) [inline sdk/evaluation/azure-ai-evaluation/README.md:26]: should we call out the need to have an Azure OpenAI/OpenAI model deployment if using GPT-based evaluators or an Azure AI Project if using content safety evaluators? Following this example here: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/storage/azure-storage-blob/README.md?plain=1#L24C1-L25C119

File: sdk/evaluation/azure-ai-evaluation/README.md [hunk 12]
```diff
@@ -455,7 +525,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
 
 <!-- LINKS -->
 
-[source_code]: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/evaluation/azure-ai-evaluation
+[source_code]: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/evaluation/azure-ai-evaluation/evaluate
```
Comment (needuv) [inline sdk/evaluation/azure-ai-evaluation/README.md:528]: can we revert this link back to the original? This link doesn't exist

**Review by ninghu — APPROVED** [review]

**Pull request merged**